third_party/ibis/ibis_oracle/__init__.py

```diff
@@ -163,3 +163,25 @@ def list_primary_key_columns(self, database: str, table: str) -> list:
                 list_pk_col_sql, parameters=(database.upper(), table.upper())
             )
             return [_[0] for _ in result.cursor.fetchall()]
+
+    def raw_metadata(
```

Comment:
```suggestion
    def raw_column_metadata(
```
A more descriptive name to differentiate from `def _metadata(self, query)` .

Probably it'd be good to add a quick documentation as well on how this method works. Not clear to me how `"SELECT * FROM {source} t0 WHERE ROWNUM <= 1"` is able to return the column and its data type 🤔

Reply:
Done, thanks